### PR TITLE
Issue 3: Support absolute path

### PR DESCRIPTION
**Pull request by stephanpelikan**
see https://github.com/pveyes/asset-graph-webpack-plugin/issues/3

### CODE REVIEW

File: lib/AssetGraphPlugin.js [hunk 1]
```diff
@@ -12,8 +12,12 @@ const path = require('path');
  */
 class AssetGraphPlugin {
   constructor(target) {
-    // get output path, relative from webpack.config.js file
-    this.output = path.join(process.cwd(), target);
+	if (path.isAbsolute(target)) {
```
Comment (pveyes) [inline lib/AssetGraphPlugin.js]: We can simply use `path.resolve` here, it will prioritize absolute path

```js
this.output = path.resolve(process.cwd(), target)
```

if target is absolute path, it will return that, if not, it will return similar to `path.join` so no need for if `path.isAbsolute`

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "asset-graph-webpack-plugin",
-  "version": "0.1.3",
+  "version": "0.1.4",
```
Comment (pveyes) [inline package.json]: Please revert this change

This will be done automatically via `npm version` command

**Review by pveyes — CHANGES_REQUESTED** [review]
Thanks for making this PR

I have a few suggestions about your changes

**Review by pveyes — APPROVED** [review]
LGTM thanks

**Comment by stephanpelikan** [comment]
I changed it according your suggestions. I also change the comment to reflect the new behaviour.
